net/wireshark4/Portfile

```diff
@@ -30,12 +30,12 @@ if { ${os.platform} eq "darwin" && ${os.major} < 16 } {
     }
 }
 
-version         4.0.6
+version         4.0.7
 revision        0
 
-checksums       sha256  0079097a1b17ebc7250a73563f984c13327dac5016b7d53165810fbcca4bd884 \
-                sha1    a60b6f8063df2a711932ba869ae86e6476087cf0 \
-                size    41583088
+checksums       sha256  a79f7b04cbff823e30452abf4bcb86773d8583eb62d5f71f16c09f019f8a8777 \
+                sha1    ad2aa262e3376d72dfe1311530dee53a8c462c1b \
```

Comment:
rmd160?

Reply:
Upstream provides size, sha256, and sha1 - just mirroring that.